src/runtime/graph/graph_runtime.cc

```diff
@@ -244,6 +250,43 @@ void GraphRuntime::ShareParams(const GraphRuntime& other, dmlc::Stream* strm) {
   this->SetupOpExecs();
 }
 
+void GraphRuntime::LinkedNDArrayDeleter(Object* container) {
+  // container is the NDArray::Container which needs to get deleted.
+  // The data member points to global const memory, so it does not need deleting.
+  delete reinterpret_cast<NDArray::Container*>(container);
```

Comment:
Should be static cast instead, since NDArray::Container* is a subclass of Object*, this can cause bug when Container and Object have a different offset

Reply:
ah, thanks. done.